diff --git a/dist/index.js b/dist/index.js
index 47e2924eb269b2161b9350b476b4bb881b635394..a9a6fc39e0b6c45bff0fb8d39cfd9e60097801a3 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2039,7 +2039,7 @@ const buildCfStyles = (values) => {
         lineHeight: values.cfLineHeight,
         letterSpacing: values.cfLetterSpacing,
         color: values.cfTextColor,
-        textAlign: values.cfTextAlign,
+        textAlign: values.cfTextAlign === 'left' ? 'start' : values.cfTextAlign === 'right' ? 'end' : values.cfTextAlign,
         textTransform: values.cfTextTransform,
         objectFit: values.cfImageOptions?.objectFit,
         objectPosition: values.cfImageOptions?.objectPosition,
